lbuild

```diff
@@ -51,7 +54,11 @@ def process(tex):
 
         print("Now compiling {}".format(tex))
 
-        ret = subprocess.run(cmd, timeout=300, stdin=DEVNULL, stdout=DEVNULL, stderr=DEVNULL)
+        try:
+            ret = subprocess.run(cmd, timeout=timeout, stdin=DEVNULL, stdout=DEVNULL, stderr=DEVNULL)
+        except subprocess.TimeoutExpired:
+            print("Timeout exceeded!")
```

Comment:
Because you catch all exceptions in the main function and I wanted to present the user some kind of error message